templates/controllers/modals/reviewRound/reviewRound.tpl

```diff
@@ -0,0 +1,89 @@
+{**
+ * templates/controllers/modals/reviewRound/reviewRound.tpl
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2003-2021 John Willinsky
+ * Copyright (c) 2021 Université Laval
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * Display reviewer review round info modal.
+ *}
+
+<script type="text/javascript">
+    // Attach the JS file tab handler.
+    $(function() {ldelim}
+        $('#historyTab').pkpHandler('$.pkp.controllers.TabHandler');
+		{rdelim});
+</script>
+<script type="text/javascript">
+    $(function() {ldelim}
+        // Attach the form handler.
+        $('#okButtonForm').pkpHandler('$.pkp.controllers.form.AjaxFormHandler');
+		{rdelim});
+</script>
+<div id="historyTab">
```

Comment:
Some styling isn't aligned, like font size, padding/margin... Maybe using `FormBuilderVocabulary` can help? @jardakotesovec, what do you think?